typescript/rekognition-lambda-s3-trigger/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "rekognition-lambda-s3-trigger",
+  "version": "1.0.0",
+  "bin": {
+    "rekognition-lambda-s3-trigger": "bin/rekognition-lambda-s3-trigger.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "*",
+    "@types/node": "10.17.27",
+    "aws-cdk": "*",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.8.0",
+    "aws-sdk": "^2.1062.0",
+    "source-map-support": "^0.5.16",
+    "constructs": "^10.0.0"
```

```suggestion
    "@types/node": "10.17.27",
    "aws-cdk": "*",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "*",
    "aws-sdk": "^2.1062.0",
    "source-map-support": "^0.5.16",
    "constructs": "*"
```

Sorry for not mentioning this in my last review but `"@aws-cdk/assert"` is actually not needed anymore, instead users should use `aws-cdk-lib/aws-assertions`. Also I changed `aws-cdk-lib` and `constructs` to have the `*` version which matches our other projects.